content/docs/csidriver/installation/operator/_index.md

```diff
@@ -19,21 +19,21 @@ If you have installed an old version of the `dell-csi-operator` which was availa
 #### Full list of CSI Drivers and versions supported by the Dell CSI Operator
 | CSI Driver         | Version   | ConfigVersion  | Kubernetes Version   | OpenShift Version     |
 | ------------------ | --------- | -------------- | -------------------- | --------------------- |
-| CSI PowerMax       | 1.6       | v5             | 1.18, 1.19, 1.20     | 4.6, 4.7              |
 | CSI PowerMax       | 1.7       | v6             | 1.19, 1.20, 1.21     | 4.6, 4.7              |
 | CSI PowerMax       | 2.0.0     | v2.0.0         | 1.20, 1.21, 1.22     | 4.6 EUS, 4.7, 4.8     |
-| CSI PowerFlex      | 1.4       | v4             | 1.18, 1.19, 1.20     | 4.6, 4.7              |
+| CSI PowerMax       | 2.1.0     | v2.1.0         | 1.20, 1.21, 1.22     | 4.8, 4.9              |
```

Comment:
@panigs7 - Openshift version should be **4.8, 4.8(EUS) and 4.9**

Reply:
Done